src/rest_api/classes/gene/widgets/phenotype.clj

```diff
@@ -510,6 +522,36 @@
                     (evidence-paper paper)))}
                (var-evidence holder rnai pheno))})))))})
 
+(defn- phenotype-table-overexpressed [db gene]
+  (let [trans-phenos (into {} (d/q
+                                q-gene-construct-transgene-pheno
+                                db gene))]
+    (->>
```

Comment:
Also think this code would read nicer (and enable reducing number of lines) with shorter function names?